sample-apps/java-sample-app/base/src/main/java/software/amazon/adot/sampleapp/RandomMetricsProducer.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package software.amazon.adot.sampleapp;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.LongCounter;
+import io.opentelemetry.api.metrics.Meter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.concurrent.TimeUnit;
+
+public class RandomMetricsProducer {
+    private static final Logger logger = LogManager.getLogger();
+
+    static int currentTimeAlive = 0;
+    static int currentActiveThreads = 0;
+
+    static String API_RANDOM_COUNTER_METRIC = "timeAlive";
+    static String API_RANDOM_ASYNC_UPDOWN_METRIC = "totalHeapSize";
+    static String API_RANDOM_UPDOWN_METRIC = "threadsActive";
+    static String API_RANDOM_GAUGE = "cpuUsage";
+
+
+    static private final Attributes COMMON_RANDOM_METRIC_ATTRIBUTES = Attributes.of(
+            AttributeKey.stringKey("signal"), "metric",
+            AttributeKey.stringKey("language"), "java",
+            AttributeKey.stringKey("metricType"), "random");
+
+    LongCounter timeCounter;
+    LongCounter threadsCounter;
+
+    long totalHeapSize;
+    long cpuUsage;
+
+    private final Config config;
+
+    static ScheduledThreadPoolExecutor threadPool = new ScheduledThreadPoolExecutor(1);
+
+    public RandomMetricsProducer(Config cfg) {
+        this.config = cfg;
+        Meter meter =
+                GlobalOpenTelemetry.meterBuilder("aws-otel")
```

Comment:
```suggestion
                GlobalOpenTelemetry.meterBuilder("aws-otel")
```
nit: Can we use a different term other than `aws-otel`? Either `ADOT` or some form of `ADOT`.